src/components/ClusterNodes/clusterNodes.scss

```diff
@@ -0,0 +1,221 @@
+.cluster-list {
+    &.no-result-container {
+        height: calc(100vh - 48px);
+        overflow: hidden;
+    }
+    .cluster-list-row {
+        display: grid;
+        grid-template-columns: auto 150px 50px 90px 100px 100px 120px;
+        column-gap: 16px;
+        .cluster-status {
+            height: 12px;
+            width: 12px;
+        }
+    }
+}
+.node-list {
+    .width-50 {
+        width: 50%;
+    }
+    .width-25 {
+        width: 25%;
+    }
+    .list-min-height {
+        min-height: calc(100vh - 290px);
+        &.no-result-container {
+            overflow: hidden;
+            display: grid;
+            grid-template-rows: 32px;
+        }
+    }
+    .search-wrapper {
+        display: grid;
+        grid-template-columns: auto 160px 1px 180px;
+        column-gap: 8px;
+
+        .outline-none {
+            outline: none !important;
+        }
+        .bottom-2px {
+            bottom: 2px;
+        }
+        .search-popup {
+            z-index: 6;
+            .search-title {
+                background: var(--window-bg);
+            }
+        }
+    }
+
+    .setting-icon path {
+        stroke: var(--N600);
+    }
+    .node-list-row {
+        display: grid;
+        grid-template-columns: auto 100px 60px 50px 100px 50px 60px 100px 100px 60px;
+        column-gap: 16px;
+    }
+
+    .sticky-column {
+        left: 0;
+        z-index: 1;
+        background-color: var(--N0);
+    }
+}
+
+.node-details-container {
+    .code-editor__header {
+        height: 32px;
+    }
+    .w-49-per {
+        width: 49%;
+    }
+    .w-51-per {
+        width: 51%;
+    }
+    .error-icon-white {
+        path[fill='#F33E3E'] {
+            fill: var(--N0);
+        }
+        path[fill='#FFF'] {
+            fill: var(--R500);
+        }
+    }
+    .alert-icon-white {
+        use {
+            fill: var(--N0);
+        }
+        path {
+            fill: var(--y500);
+        }
+    }
+    .no-data-icon {
+        path[fill='#06C'] {
+            fill: var(--N600);
+        }
+    }
+    .no-data-tab {
+        padding-top: 105px;
+        padding-bottom: 105px;
+    }
+
+    .hover-only {
+        display: none;
+    }
+
+    .hover-trigger {
+        .tippy-box.default-tt {
+            display: none !important;
+        }
+        &:hover {
+            .tippy-box.default-tt {
+                display: block !important;
+            }
+            .hover-only {
+                display: inline-block;
+            }
+        }
+    }
+    .subtab-grid {
+        display: grid;
+        grid-template-columns: 50% 50%;
+    }
+    .node-details-grid {
+        display: grid;
+        grid-template-columns: 300px calc(100vw - 415px);
+        column-gap: 12px;
+        .resource-row {
+            display: grid;
+            grid-template-columns: 20px minmax(auto, 200px) 100px 100px 100px 100px 100px;
```

Comment:
it can also be written as - `20px minmax(auto, 200px) repeat(5, 100px)`

Reply:
done